Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM golang:1.9-alpine as builder
+FROM golang:1.14-alpine as builder
 
 COPY . /go/src/mumble.info/grumble
 
 WORKDIR /go/src/mumble.info/grumble
 
-RUN apk add --no-cache git \
-  && go get -v -t ./... \
+RUN apk add --no-cache git build-base
```

Comment:
How come `build-base` is now required/used instead?

Reply:
It looks like in the golang:1.14-alpine container cgo is required to build a module we depend on. It's possible in the past they included gcc and tools in the container by default, but it looks like they do not now.

`# runtime/cgo
exec: "gcc": executable file not found in $PATH
`